ipatests/prci_definitions/nightly_master.yaml

```diff
@@ -75,6 +75,18 @@ jobs:
         timeout: 3600
         topology: *master_1repl
 
+  fedora-30/external_ca_templates:
+    requires: [fedora-30/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{fedora-30/build_url}'
+        test_suite: test_integration/test_external_ca.py::TestExternalCAProfileScenarios
+        template: *ci-master-f30
+        timeout: 3600
+        topology: *master_1repl
+
```

Comment:
@flo-renaud no worries, PR updated.  I included `nightly_master_pki` too, because this feature does depend on some less-frequently-used `pki_default.cfg(5)` knobs.